packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list/entity-type-item.tsx

```diff
@@ -96,7 +98,7 @@ export const EntityTypeItem: VFC<EntityTypeItemProps> = ({
           size="medium"
           unpadded
           sx={({ palette }) => ({
-            color: palette.gray[40],
+            color: "transparent",
```

Comment:
We should add the color back in `:focus-visible`, so it doesn't look like this when tabbed into

<img width="424" alt="Screenshot 2022-06-17 at 16 03 32" src="https://user-images.githubusercontent.com/16928688/174325341-5b4adc98-f5fa-49ed-b77e-4cb103e57c6c.png">

Reply:
Good catch - all fixed now